huaweicloud/services/cdn/resource_huaweicloud_cdn_domain.go

```diff
@@ -1068,6 +1072,19 @@ func resourceCdnDomainV1Update(ctx context.Context, d *schema.ResourceData, meta
 	id := d.Id()
 	opts := getResourceExtensionOpts(d, cfg)
 	timeout := d.Timeout(schema.TimeoutCreate)
+	status := true
+	if d.Get("private_bucket_access").(bool) {
+		privateBucketAccessOpts := &domains.PrivateBucketAccessOpts{
+			Status: &status,
+		}
+		privateBucketAccess, err := domains.UpdatePrivateBucketAccess(cdnClient, d.Id(), privateBucketAccessOpts).Extract()
```

Comment:
1. how can the user disable private bucket access?
2. please add an item before the resource because we have called a new API;
3. if other fields changes, like tags or sources, it will also call the API to update private bucket access, please check it.